### PR TITLE
Hotfix/ch227 - Fix event to drop all fields empty into the smart contract template 

### PR DESCRIPTION
**Pull request by 92Sam**
https://app.clubhouse.io/masterventures/story/227 - Task Comment 

:white_check_mark: Fix - agreement content needs to be aligned with the provided doc
image

:white_check_mark: Fix - clearing the placeholders in the agreement if the user does not enter any data for the specific field is required

### CODE REVIEW

File: redux/actions/template/index.ts [hunk 1]
```diff
@@ -1210,14 +1206,51 @@ const getContractTemplate = (
...
+          'address',
+          'aba',
+          'payeeAccount',
+          'payeeAccountName',
+          
+          'ethereum',
+
+          'bitcoin',
+          'state',
+          'jurisdiction',
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts:1245]: Not needed for this PR, but consider using a enum for these.

File: redux/actions/template/index.ts [hunk 2]
```diff
@@ -28,6 +28,7 @@ const getContractTemplate = (
   contractName: String,
   isEditing?: Boolean,
   agreementReviewed?: Boolean,
+  isHiddenEmpty?: Boolean,
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts:31]: what is the purpose of this variable?

File: pages/new-agreement.tsx [hunk 3]
```diff
@@ -223,6 +237,8 @@ const NewAgreement: NextPage<NewAgreementProps> = ({ templateTypeCode }) => {
       while (!cid) {
         cid = await toIpfs();
       }
+      console.log("ON SEND FORM")
+      console.log(jsonSchemas)
```
Comment (CoolHandLuke88) [inline pages/new-agreement.tsx:241]: remove console log

File: redux/reducers/smartAgreements.ts [hunk 4]
```diff
@@ -150,34 +150,41 @@ const smartAgreementsReducer = (
...
     case SmartAgreementsTypes.SET_SAFT_AGEEMENT_DATA: {
-      const newSaftData = payload;
+      const newSaftData = payload
+      // console.log(newSaftData)
+      // if (!newSaftData.ethereum) {
+      //   newSaftData.ethereum = ' '
+      // }
+      // if (!newSaftData.bitcoin) {
+      //   newSaftData.bitcoin = ' '
+      // }
```
Comment (CoolHandLuke88) [inline redux/reducers/smartAgreements.ts:173]: remove dead code

**Comment by CoolHandLuke88** [comment]
see comments

File: redux/actions/template/index.ts (hunk 1 above)
Comment (92Sam) [reply redux/actions/template/index.ts:1245]: yes my friend we need centralize all this in common fields for all templates. we need modify this.

File: pages/new-agreement.tsx (hunk 3 above)
Comment (92Sam) [reply pages/new-agreement.tsx:241]: droping this thanks

File: redux/reducers/smartAgreements.ts (hunk 4 above)
Comment (92Sam) [reply redux/reducers/smartAgreements.ts:173]: dropping code.

File: redux/actions/template/index.ts (hunk 2 above)
Comment (92Sam) [reply redux/actions/template/index.ts:31]: We need drop this actually we are not using this.

**Comment by 92Sam** [comment]
@CoolHandLuke88 code cleaned my friend, ready to review again.

**Review by CoolHandLuke88 — APPROVED** [review]
ty @92Sam!

**Pull request merged**